### PR TITLE
use the proper enums for palette loading

### PR DESCRIPTION
**Pull request by red031000**
there is still some functions to do with palette loading that do not use the enums, however, I cannot tell whether they *should* use the enums or not

### CODE REVIEW

File: src/battle_arcade_game_board.c [hunk 1]
```diff
@@ -568,8 +568,8 @@ static void BattleArcadeGameBoard_InitObjects(GAME_BOARD_WORK *work) {
         work->unk78[i] = String_New(32, HEAP_ID_GAME_BOARD);
     }
 
-    LoadFontPal0(GF_BG_LYR_MAIN_0, 416, HEAP_ID_GAME_BOARD);
-    LoadFontPal1(GF_BG_LYR_MAIN_0, 384, HEAP_ID_GAME_BOARD);
+    LoadFontPal0(GF_PAL_LOCATION_MAIN_BG, GF_PAL_SLOT_OFFSET_13, HEAP_ID_GAME_BOARD);
```
Comment (adrienntindall) [inline src/battle_arcade_game_board.c:571]: Question- does GF_PAL_SLOT_OFFSET_X mean the offset to the Xth indexed palette? Ie this function is loading in palette data to palette 13?

Comment (red031000) [reply src/battle_arcade_game_board.c:571]: each palette slot is 0x20 in size. offset 13 loads into the 13th palette (zero indexed)

Comment (red031000) [reply src/battle_arcade_game_board.c:571]: (for context, slot 13 loads at 0x1A0)

Comment (adrienntindall) [reply src/battle_arcade_game_board.c:571]: Would something like PAL_SLOT_13_OFFSET be more clear then?

Comment (red031000) [reply src/battle_arcade_game_board.c:571]: GF_PAL_SLOT_13_OFFSET? could possibly work

**Comment by red031000** [comment]
updated

**Review by adrienntindall — APPROVED** [review]

**Pull request merged**